cmd/kubeapps-apis/server/packages.go

```diff
@@ -118,6 +161,125 @@ func (s packagesServer) GetAvailablePackageDetail(ctx context.Context, request *
 	}, nil
 }
 
+// GetInstalledPackageSummaries returns the installed package summaries based on the request.
+func (s packagesServer) GetInstalledPackageSummaries(ctx context.Context, request *packages.GetInstalledPackageSummariesRequest) (*packages.GetInstalledPackageSummariesResponse, error) {
+	contextMsg := ""
+	if request.Context != nil {
+		contextMsg = fmt.Sprintf("(cluster=[%s], namespace=[%s])", request.Context.Cluster, request.Context.Namespace)
+	}
```

Comment:
You could just replace the above 4 lines with:
```
contextMsg = fmt.Sprintf("(cluster=[%s], namespace=[%s])", request.GetContext().GetCluster(), request.GetContext().GetNamespace())
```

if you're OK with "(cluster=[], namespace=[])" when they're not defined.

Reply:
Great, this also applies to the rest of the helm/kapp already implemented operations. I'll send an additional PR for that.